<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="2" column="0">
     <widget class="QFrame" name="frame_4">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>189</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_agregar_datos_factura">
         <property name="minimumSize">
          <size>
           <width>125</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>125</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 1px solid #ccc;
    border-radius: 10px;           /* Bordes redondeados con radio de 10 píxeles */
    padding: 5px;
	font: bold 9pt &quot;Nunito&quot;;                  /* Espaciado interno del botón */
}

QPushButton:hover{
font: bold 9pt &quot;Nunito&quot;;
background-color: #f0f0f0; /* Color de fondo deseado */
}
</string>
         </property>
         <property name="text">
          <string>Agregar</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>InterfaceImages/lista-de-verificacion (1).png</normaloff>InterfaceImages/lista-de-verificacion (1).png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_8">
         <property name="minimumSize">
          <size>
           <width>125</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>125</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 1px solid #ccc;
    border-radius: 10px;           /* Bordes redondeados con radio de 10 píxeles */
    padding: 5px;
	font: bold 9pt &quot;Nunito&quot;;                  /* Espaciado interno del botón */
}

QPushButton:hover{
font: bold 9pt &quot;Nunito&quot;;
background-color: #f0f0f0; /* Color de fondo deseado */
}
</string>
         </property>
         <property name="text">
          <string>Cancelar</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>InterfaceImages/eliminar.png</normaloff>InterfaceImages/eliminar.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="frame_3">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="frame_6">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="leftMargin">
           <number>50</number>
          </property>
          <property name="rightMargin">
           <number>50</number>
          </property>
          <item>
           <widget class="QLabel" name="label_13">
            <property name="minimumSize">
             <size>
              <width>125</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
font: bold 9pt &quot;Nunito&quot;;</string>
            </property>
            <property name="text">
             <string>Nombre:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_agNombrePaciente">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
    background-color: #f0f0f0; /* Color de fondo deseado */
    border: 1px solid #ccc; /* Color del borde deseado */
    padding: 5px; /* Espacio alrededor del texto */
    color: rgb(0, 0, 0); /* Color del texto deseado */
  	font: 9pt &quot;Nunito&quot;; /* Fuente deseada */
    font-size: 14px; /* Tamaño de fuente deseado */
    border-radius: 8px; /* Redondear las esquinas */
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_11">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>50</number>
          </property>
          <property name="rightMargin">
           <number>50</number>
          </property>
          <item>
           <widget class="QLabel" name="label_18">
            <property name="minimumSize">
             <size>
              <width>125</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
font: bold 9pt &quot;Nunito&quot;;</string>
            </property>
            <property name="text">
             <string>Teléfono:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_agApellido">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
    background-color: #f0f0f0; /* Color de fondo deseado */
    border: 1px solid #ccc; /* Color del borde deseado */
    padding: 5px; /* Espacio alrededor del texto */
    color: rgb(0, 0, 0); /* Color del texto deseado */
  	font: 9pt &quot;Nunito&quot;; /* Fuente deseada */
    font-size: 14px; /* Tamaño de fuente deseado */
    border-radius: 8px; /* Redondear las esquinas */
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_5">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLineEdit" name="le_NIT">
          <property name="geometry">
           <rect>
            <x>320</x>
            <y>10</y>
            <width>250</width>
            <height>35</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
    background-color: #f0f0f0; /* Color de fondo deseado */
    border: 1px solid #ccc; /* Color del borde deseado */
    padding: 5px; /* Espacio alrededor del texto */
    color: rgb(0, 0, 0); /* Color del texto deseado */
  	font: 9pt &quot;Nunito&quot;; /* Fuente deseada */
    font-size: 14px; /* Tamaño de fuente deseado */
    border-radius: 8px; /* Redondear las esquinas */
}
</string>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>20</y>
            <width>125</width>
            <height>16</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>125</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);
font: bold 9pt &quot;Nunito&quot;;</string>
          </property>
          <property name="text">
           <string>NIT:</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_10">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="leftMargin">
           <number>50</number>
          </property>
          <property name="rightMargin">
           <number>50</number>
          </property>
          <item>
           <widget class="QLabel" name="label_17">
            <property name="minimumSize">
             <size>
              <width>125</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
font: bold 9pt &quot;Nunito&quot;;</string>
            </property>
            <property name="text">
             <string>Dirección:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>34</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="le_agDpi_2">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
    background-color: #f0f0f0; /* Color de fondo deseado */
    border: 1px solid #ccc; /* Color del borde deseado */
    padding: 5px; /* Espacio alrededor del texto */
    color: rgb(0, 0, 0); /* Color del texto deseado */
  	font: 9pt &quot;Nunito&quot;; /* Fuente deseada */
    font-size: 14px; /* Tamaño de fuente deseado */
    border-radius: 8px; /* Redondear las esquinas */
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_8">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label_14">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>600</width>
            <height>40</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);
font: bold 12pt &quot;Nunito&quot;;</string>
          </property>
          <property name="text">
           <string>Información del comision</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_7">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QComboBox" name="box_doc">
          <property name="geometry">
           <rect>
            <x>330</x>
            <y>20</y>
            <width>221</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>20</y>
            <width>101</width>
            <height>21</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);
font: bold 9pt &quot;Nunito&quot;;</string>
          </property>
          <property name="text">
           <string>Lo vendio</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>125</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>650</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
font: bold 12pt &quot;Nunito&quot;;</string>
            </property>
            <property name="text">
             <string>Información del Cliente</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
